assets/css/main.css

```diff
@@ -836,3 +836,7 @@ sl-tab-group {
   border-color: var(--sl-color-gray-200);
   color: var(--sl-color-gray-800);
 }
+
+a {
```

Comment:
so you disable the text decoration for links globally? Isn't that going to affect how links are displayed?

Reply:
You're right, too dangerous to do that right now, let's keep that for later